Hearthstone Deck Tracker/Controls/Overlay/Battlegrounds/HeroPicking/BattlegroundsHeroPickingViewModel.cs

```diff
@@ -14,16 +18,32 @@ public class BattlegroundsHeroPickingViewModel : ViewModel
 	{
 		public Visibility Visibility
 		{
-			get
+			get => GetProp(Collapsed);
+			set
 			{
-				if(!Config.Instance.ShowBattlegroundsHeroPicking)
-					return Collapsed;
-				return GetProp(Collapsed);
+				SetProp(value);
+				if(value == Visible)
+					StatsVisibility = Config.Instance.ShowBattlegroundsHeroPicking ? Visible : Collapsed;
```

Comment:
You can bind this directly in xaml, see e.g. https://github.com/HearthSim/Hearthstone-Deck-Tracker/blob/38aa73a7d8d8895f682132ba6187380841968733/Hearthstone%20Deck%20Tracker/Controls/DeckPicker/DeckPickerItemLayouts/DeckPickerItemLayout1.xaml#L52